Formula/harmony-one.rb

```diff
@@ -0,0 +1,79 @@
+class HarmonyOne < Formula
+  desc "CLI tool to interact with the Harmony Blockchain"
+  homepage "https://docs.harmony.one/"
+  url "https://github.com/harmony-one/harmony/archive/refs/tags/v4.2.1.tar.gz"
+  sha256 "d29609a99a3a2e9031d2a26414ffdc3e271c3bf9cf9331dfcb321715fef682d4"
+  license "LGPL-3.0-or-later"
+
+  depends_on "go@1.16" => :build
+
+  depends_on arch: :x86_64
```

Comment:
Why doesn't it work on new macs?

Reply:
I will try to debug this after I solved the linux build issues. It was failing in the CI and I can't test it without M1 device. :)